ground/src/main/java/com/google/android/ground/repository/LocationOfInterestRepository.kt

```diff
@@ -52,10 +53,15 @@ constructor(
   private val remoteDataStore: RemoteDataStore,
   private val mutationSyncWorkManager: MutationSyncWorkManager,
   private val uuidGenerator: OfflineUuidGenerator,
+  private val authenticationManager: AuthenticationManager
 ) {
   /** Mirrors locations of interest in the specified survey from the remote db into the local db. */
   suspend fun syncLocationsOfInterest(survey: Survey) {
-    val lois = remoteDataStore.loadLocationsOfInterest(survey)
+    val creatorEmail = authenticationManager.getAuthenticatedUser().email
```

Comment:
Can we add a TODO here to add some check in the future to control this behavior?

Reply:
Sure. It's an FR, not a bug, but I'll add one for posterity.